doxia-modules/doxia-module-markdown/src/main/java/org/apache/maven/doxia/module/markdown/MarkdownMarkup.java

```diff
@@ -77,11 +77,11 @@ public interface MarkdownMarkup extends TextMarkup {
     /** Syntax for the link start: "](" */
     String LINK_START_2_MARKUP = "](";
 
-    /** Syntax for the ordered list item: '1.' */
-    String LIST_ORDERED_ITEM_START_MARKUP = "1.";
+    /** Syntax for the ordered list item: '1. ' */
+    String LIST_ORDERED_ITEM_START_MARKUP = "1. ";
 
-    /** Syntax for the unordered list item: "-" */
-    String LIST_UNORDERED_ITEM_START_MARKUP = "-";
+    /** Syntax for the unordered list item: "- " */
+    String LIST_UNORDERED_ITEM_START_MARKUP = "- ";
```

Comment:
This is rather a fix, right?

Reply:
Previously this was combined each time with a space, but it never occurs without it, so it makes sense to include the space in the constant.

Reply:
Makes sense.